src/lib-components/BlockStaffList.vue

```diff
@@ -76,6 +76,14 @@ export default {
             type: String,
             default: "",
         },
+        alternativeFullName: {
```

Comment:
I don't think that we will need alternativeFullName.

Reply:
yes, we should remove the props if they are not used.